src/test/scala/org/broadinstitute/dsde/workbench/sam/service/UserServiceSpecs/CreateUserSpec.scala

```diff
@@ -27,71 +20,42 @@ class CreateUserSpec extends UserServiceTestTraits {
 
-  describe("UserService.createUser") {
-    describe("returns an enabled UserStatus that has NOT accepted ToS") {
-      it("when user has an AzureB2CId and a GoogleSubjectId") {
-        // Setup
-        val userWithBothIds = genWorkbenchUserBoth.sample.get
-        val userService = TestUserServiceBuilder()
-          .withAllUsersGroup(allUsersGroup)
-          .build
-
-        // Act
-        val userStatus = runAndWait(userService.createUser(userWithBothIds, samRequestContext))
-
-        // Assert
-        inside(userStatus) { status =>
-          status should beForUser(userWithBothIds)
-          "google" should beEnabledIn(status)
-          "ldap" should beEnabledIn(status)
-          "allUsersGroup" should beEnabledIn(status)
-          "adminEnabled" should beEnabledIn(status)
-          "tosAccepted" shouldNot beEnabledIn(status)
-        }
-      }
-
-      it("when user has an AzureB2CId but no GoogleSubjectId") {
-        // Setup
-        val userWithOnlyB2CId = genWorkbenchUserAzure.sample.get
-        val userService = TestUserServiceBuilder()
-          .withAllUsersGroup(allUsersGroup)
-          .build
-
-        // Act
-        val userStatus = runAndWait(userService.createUser(userWithOnlyB2CId, samRequestContext))
-
-        // Assert
-        inside(userStatus) { status =>
-          status should beForUser(userWithOnlyB2CId)
-          "google" should beEnabledIn(status)
-          "ldap" should beEnabledIn(status)
-          "allUsersGroup" should beEnabledIn(status)
-          "adminEnabled" should beEnabledIn(status)
-          "tosAccepted" shouldNot beEnabledIn(status)
-        }
-      }
-
-      it("when user has GoogleSubjectId but no AzureB2CId") {
-        // Setup
-        val userWithOnlyGoogleId = genWorkbenchUserGoogle.sample.get
-        val userService = TestUserServiceBuilder()
-          .withAllUsersGroup(allUsersGroup)
-          .build
-
-        // Act
-        val userStatus = runAndWait(userService.createUser(userWithOnlyGoogleId, samRequestContext))
-
-        // Assert
-        inside(userStatus) { status =>
-          status should beForUser(userWithOnlyGoogleId)
-          "google" should beEnabledIn(status)
-          "ldap" should beEnabledIn(status)
-          "allUsersGroup" should beEnabledIn(status)
-          "adminEnabled" should beEnabledIn(status)
-          "tosAccepted" shouldNot beEnabledIn(status)
+  val userWithBoth = (genWorkbenchUserBoth.sample.get, "user with google and azure ids")
+  val userWithGoogle = (genWorkbenchUserGoogle.sample.get, "user with google id")
+  val userWithAzure = (genWorkbenchUserAzure.sample.get, "user with azure id")
+  val newUsers = Vector(userWithBoth, userWithGoogle, userWithAzure)
+
+
+  describe("when a new user is created") {
+    describe("that does not already exist") {
+      newUsers.foreach { userTuple =>
+        val (newUser, description) = userTuple
+        describe(s"UserService.createUser with a $description") {
```

Comment:
Nit: can we reword this message a tiny bit? I find the phrasing of "when a new user… that does not already exist… Userservice.createUser with a user with google and azure ids" more confusing than it was before

Reply:
I think it needs one additional verb or something